src/eth_swap_utils.h

```diff
@@ -18,9 +18,24 @@
 #pragma once
 
 #include <stdint.h>
+#include "swap_lib_calls.h"
+#include "chainConfig.h"
+#include "caller_api.h"
 
-bool parse_swap_config(const uint8_t* config,
+typedef struct eth_libargs_s {
```

Comment:
Why is the definition of this struct duplicated from eth_swap_lib_calls.h ?

Reply:
Good catch. Copy-Past error :wink: